### ISSUE TITLE
FR: Support for networks without encryption

### ISSUE THREAD

**Issue by retrue** [koreader/koreader#2784]
* KOReader version: Last nightlies
* Device: Kobo Mini

#### Issue
This morning I tried to connect to a public WiFi (the public library of my town). It doesn't need password. And I got this error message:
_Networks without WPA/WPA2 encryption are not supported._
Would it be possible to add support for not encrypted Wi-Fi networks?

**Comment by Frenzie** [comment]
There's already a todo marker: https://github.com/koreader/koreader/blob/3e656d71e07bc4460ee02cbf8aed5a6fd36389c7/frontend/ui/network/wpa_supplicant.lua#L55 ;-)

On your computer you can just say

```
network={
    ssid="somethingsomething"
    key_mgmt=NONE
}
```

**Comment by Frenzie** [comment]
I noticed this is also mentioned in #2183. Not closing this one because it's clearer to track this separately; just mentioning it to link the issues.

**Comment by Frenzie** [comment]
Implemented in #8196.
